modules/custom/d_update/d_update.module

```diff
@@ -72,15 +72,13 @@ function d_update_preprocess_menu__toolbar(&$variables) {
   if (empty($variables['items']['d_update.admin_update'])) {
     return;
   }
-  if (d_update_update_link_is_active()) {
-    if (d_update_system_has_pending_updates()) {
-      // In this stage title is already a string, so we need to call __toString.
-      $variables['items']['d_update.admin_update']['title'] = t('Droopler Update')->__toString();
-      // We need to rewrite current url options to hold classes etc.
-      $options = $variables['items']['d_update.admin_update']['url']->getOptions();
-      $variables['items']['d_update.admin_update']['url'] = Url::fromRoute('system.db_update')
-        ->setOptions($options);
-    }
+  if (d_update_checklist_has_pending_updates()) {
```

Comment:
in droopler update button context we do not need check system updates, so d_update_system_has_pending_updates is removed from condition